modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeCatchAllCheckTest.java

```diff
@@ -125,10 +124,23 @@ public void testUpgradeCatchAllCheck() throws Exception {
 			StringUtil.replace(_issueKey, CharPool.DASH, CharPool.UNDERLINE) +
 				".test" + _fileType;
 
-		if (_hasValidUpgradeFiles(_issueKey, _fileType)) {
-			_testUpgradeCatchAllCheck(
-				"upgrade/upgrade-catch-all-check/" + fileName);
-		}
+		Path filePath = Paths.get(
+			"src/test/resources/com/liferay/source/formatter/dependencies" +
+				"/upgrade/upgrade-catch-all-check/" + fileName);
+
+		Path expectedFilePath = Paths.get(
+			"src/test/resources/com/liferay/source/formatter/dependencies" +
+				"/expected/upgrade/upgrade-catch-all-check/" + fileName);
+
+		Assert.assertTrue(
+			"Missing unit test in " + filePath, Files.exists(filePath));
+
+		Assert.assertTrue(
+			"Missing unit test in " + expectedFilePath,
+			Files.exists(expectedFilePath));
```

Comment:
Let's sort the file paths and assertions "as used", meaning:

- declare `filePath`
- assert `filePath` exists, with unique message
- declare `expectedFilePath`
- assert `expectedFilePath` exists, with unique message

Make sense?